README.md

```diff
@@ -118,11 +129,41 @@ from dataverse.etl import ETLPipeline
 etl_pipeline = ETLPipeline()
 spark, dataset = etl_pipeline.run(config=ETL_config, verbose=True)
 ```
+ETLPipeline is an object designed to manage the ETL processes. By inserting `ETL_config` which is defined in the previous step into ETLpipeline object and calling the `run` method, stacked ETL blocks are sequentially executed.
```

Comment:
Can you add an explanation that more than one ETL_config can be run sequentially?